### PR TITLE
Allow the caller to pass IsCertRolloverEnabled for !net452 builds

### PR DESCRIPTION
**Pull request by manish1604**
Allow the user to pass IsCertificateRollOverEnabled flag to be passed while acquiring token.

### CODE REVIEW

File: sdk/mgmtcommon/Auth/Az.Auth/Az.Authentication/ApplicationTokenProvider.cs [hunk 1]
```diff
@@ -444,6 +444,33 @@ public static async Task<ServiceClientCredentials> LoginSilentAsync(string domai
 #endif
         }
 
+#if !net452
```
Comment (shahabhijeet) [inline sdk/mgmtcommon/Auth/Az.Auth/Az.Authentication/ApplicationTokenProvider.cs]: Is this formatting change and rearranging of regions?
Or am I missing the change here?

**Review by AlexGhiondea — APPROVED** [review]
Approved, pending the answer to @shahabhijeet's question.

**Review by AlexGhiondea — CHANGES_REQUESTED** [review]
Actually -- we need to bump the package version for this.

File: sdk/mgmtcommon/Auth/Az.Auth/Az.Authentication/ApplicationTokenProvider.cs (hunk 1 above)
Comment (shahabhijeet) [reply sdk/mgmtcommon/Auth/Az.Auth/Az.Authentication/ApplicationTokenProvider.cs]: Sorry I was not able to figure out the diff, now I can :)
There is no harm in adding an overload, but you can still achieve the same using an existing overload on line 573 (new up AuthenticationProvider with your certificate and the additional flag)
The only downside to taking this change is to release this package and then update dependent packages and release them as well.
And if you can use the overload on line 573, we can avoid releases of these packages.
I will let @AlexGhiondea decide this.

**Comment by AlexGhiondea** [comment]
@manish1604 could you take a look here and let me know if this is still needed? Thank you!

**Comment by jsquire** [comment]
@manish1604 - Would you please be so kind as to let us know if this is still an active work stream or if we should close out the PR?

**Comment by manish1604** [comment]
This can be closed.

**Pull request closed without merging**